app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java

```diff
@@ -327,6 +325,25 @@ private void postDescription(@NonNull String editToken) {
                     }));
         }
 
+        private Observable<MwPostResponse> getPostService(@NonNull String editToken, @Nullable String languageCode) {
```

Comment:
Perhaps `getPostObservable()` is a clearer name?

Reply:
Right!